### ISSUE TITLE
Interactive server capacity

### ISSUE THREAD

**Issue by antagomir** [rOpenGov/homepage#3]
We should start planning how to provide interactive blog & services through ropengov server. Perhaps picking some ideas from web resources developer by @muuankarski

**Comment by pitkant** [comment]
What type of interactive capabilities and services should be considered? Being able to host our own Shiny server, self-hosted comments...?

**Comment by antagomir** [comment]
Yes, Shiny at least, and hosting html reports, unless there are further needs / ideas. Some suitable server capacity is available.
